custom_components/mail_and_packages/const.py

```diff
@@ -347,6 +347,7 @@
         "subject": [
             "parcel is now with your local Hermes courier",
             "Ihre Hermes Sendung",
+            "Deine Hermes Sendung von OTTO ist auf dem Weg",
```

Comment:
```suggestion
            "Deine Hermes Sendung",
```

this should work better.